zitadel/default_invite_user_message_text/resource.go

```diff
@@ -0,0 +1,274 @@
+package default_invite_user_message_text
+
+import (
+	"context"
+	"fmt"
+
+	github_com_hashicorp_terraform_plugin_framework_diag "github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	github_com_hashicorp_terraform_plugin_framework_tfsdk "github.com/hashicorp/terraform-plugin-framework/tfsdk"
+	github_com_hashicorp_terraform_plugin_framework_types "github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/zitadel/zitadel-go/v3/pkg/client/zitadel/admin"
```

Comment:
What's the need for aliases this long? Could work as well with no aliases

Reply:
This seems to be a naming convention in use across the codebase since it uses both the old TF SDK and the new TF Framework SDK. 

There is a migration PR in progress which should clean this up. That's my understanding